src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -1493,15 +1493,15 @@ def create_service_principal_for_rbac(
                         logger.warning('  Retrying role assignment creation: %s/%s', retry_time + 1,
                                        _RETRY_TIMES)
                         continue
-                    elif _error_caused_by_role_assignment_exists(ex):
+                    if _error_caused_by_role_assignment_exists(ex):
                         logger.warning('  Role assignment already exists.\n')
                         break
```

Comment:
It is no-else-break.

Reply:
I know. Fixed in https://github.com/Azure/azure-cli/pull/18000.